jstests/list_movies.test.js

```diff
@@ -78,7 +78,7 @@ global.Audio = function() {
       list_movies_data(movies);
       expect($.fn.html).toHaveBeenCalledTimes(4);
   
-      const expectedEmptyHtml = expect.stringContaining('<i>No movies</i>');
+      const expectedEmptyHtml = expect.stringContaining('<table><tbody><tr><td><i>No movies</i></td></tr><tr><td colspan=\"6\"><a href=\"/upload\">Click here to upload a movie</a></td></tr></tbody></table>');
       expect($.fn.html).toHaveBeenCalledWith(expectedEmptyHtml);
```

Comment:
This seems super brittle. Shouldn't we have the string constants defined in a common file that's used in both the app and the test?

Also, shouldn't we have a function that creates that matrix, rather than repeating the code on lines 81, 148, 52, etc?

Reply:
Yes that is definitely a better method of going about this.